src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -164,34 +125,36 @@ function getRouteForCurrentStep(currentStep) {
     }
 }
 
-function ReimbursementAccountPage({reimbursementAccount, route, onfidoToken, policy, account, isLoadingApp, session, plaidLinkToken, plaidCurrentEvent, reimbursementAccountDraft}) {
+function ReimbursementAccountPage({
+    reimbursementAccount,
+    route,
+    onfidoToken = '',
+    policy,
+    account,
+    isLoadingApp = false,
+    session,
+    plaidLinkToken = '',
+    plaidCurrentEvent = '',
+}: ReimbursementAccountPageProps) {
     /**
         The SetupWithdrawalAccount flow allows us to continue the flow from various points depending on where the
         user left off. This view will refer to the achData as the single source of truth to determine which route to
         display. We can also specify a specific route to navigate to via route params when the component first
         mounts which will set the achData.currentStep after the account data is fetched and overwrite the logical
         next step.
     */
-    const achData = lodashGet(reimbursementAccount, 'achData', {});
+    const achData = reimbursementAccount?.achData;
 
-    /**
-     * @param {Array} fieldNames
-     *
-     * @returns {Object}
-     */
-    function getBankAccountFields(fieldNames) {
+    function getBankAccountFields<T extends InputID>(fieldNames: T[]): Pick<ACHData, T> {
         return {
-            ..._.pick(lodashGet(reimbursementAccount, 'achData'), ...fieldNames),
+            ...lodashPick(reimbursementAccount?.achData, ...fieldNames),
```

Comment:
Is possible to use a [native solution](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore?tab=readme-ov-file#_pick)? e.g.

```ts
// Native
const { a, c } = object;
const result = { a, c};
console.log(result);
// output: {a: 1, c: 3}
// for an array of this object --> array.map(({a, c}) => ({a, c}));
```

Reply:
hmm , I asked chatgpt about that and its providing this solution
<img width="718" alt="image" src="https://github.com/Expensify/App/assets/25230417/0245474f-54c0-453f-a088-8b44f8c92127">
which seems not worth to implement in our case